pkg/sql/alter_type.go

```diff
@@ -104,17 +107,17 @@ func (p *planner) renameType(params runParams, n *alterTypeNode, newName string)
 
 	// Rename the base descriptor.
 	if err := p.performRenameTypeDesc(
-		params.ctx,
+		ctx,
 		n.desc,
 		newName,
-		tree.AsStringWithFQNames(n.n, params.Ann()),
+		tree.AsStringWithFQNames(n.n, p.ExtendedEvalContext().EvalContext.Annotations),
```

Comment:
I'd probably just add a `p.Ann()` function that returns this for convenience

Reply:
Done